Cantera/conda-recipes
# Prevent installation of cantera from PyPI

When running `pip install` in `build.sh/.bat`, fetching `cantera` from PyPi needs to be prevented.

Closes #31

Closes #30 (via Cantera/cantera#1358)
Circumvents #29 (MKL is disabled)

Output with [manually built package](https://gist.github.com/ischoegl/b7c935bb3c94403f3f4d4e5db67ac116):
```
>>> import cantera as ct
>>> ct.__version__
'3.0.0a2'
```

Other fixes:
- Include `ck2yaml` in package
- Update `pytest`
- Update deprecated SCons flag
- Temporarily disable MKL, see #29
- Add 'official' `share/cantera/data` to `libcantera` package
